spec/views/items/index.html.erb_spec.rb

```diff
@@ -1,31 +1,14 @@
 require 'rails_helper'
 
-RSpec.describe "items/index", type: :view do
-  before(:each) do
-    assign(:items, [
-      Item.create!(
-        :service => nil,
-        :event => nil,
-        :status => 1,
-        :text => "MyText",
-        :image_link => "Image Link"
-      ),
-      Item.create!(
-        :service => nil,
-        :event => nil,
-        :status => 1,
-        :text => "MyText",
-        :image_link => "Image Link"
-      )
-    ])
-  end
+RSpec.describe 'items/index', type: :view do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.